subsys/dult/user.c

```diff
@@ -43,6 +44,12 @@ int dult_user_register(const struct dult_user *user)
 		return -EINVAL;
 	}
 
+	if (IS_ENABLED(CONFIG_DULT_MOTION_DETECTOR) &&
+	    !(user->accessory_capabilities &
+	      BIT(DULT_ACCESSORY_CAPABILITY_MOTION_DETECTOR_UT_BIT_POS))) {
+		return -EINVAL;
+	}
```

Comment:
Since the MD feature is optional, I can imagine a case where one DULT user supports the MD capability and the other one does not. Should we write this code so that the feature is active depending on the declared capability?